os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -357,10 +354,19 @@ static int loading_thread(int argc, char *argv[])
 		return -1;
 	}
 
+	/* Arguments : [1] type */
 	type = (pid_t)atoi(argv[1]);
 
 	ret = ERROR;
 	switch (type) {
+	case LOADCMD_LOAD:
+		if (argc <= 2) {
```

Comment:
You used 'argc <= 1' in L352.  Why do you use number 2 here? 
And what's meaning of '1' and '2'?

Reply:
The value means the number of arguments of loading thread.
argc is always greater than or equal to 1 because argv[0] is thread name by default.
And the this thread use more than 1 arguments 'type'.
So argc should be greater than or equal to 2.
In case of LOADCMD_LOAD, argc should be equal to 3 because 2 arguments, 'type' and 'bin_idx' are passed.